CHANGELOG.md

```diff
@@ -162,6 +162,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - [Discover] Enhanced the data source selector with added sorting functionality ([#5719](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5719))
 - [Multiple Datasource] Add datasource picker component and use it in devtools and tutorial page when multiple datasource is enabled ([#5756](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5756))
 - [Multiple Datasource] Add datasource picker to import saved object flyout when multiple data source is enabled ([#5781](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5781))
+- [Multiple Datasource] Add show default datasource for selector picker ([#6293](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6293/files))
```

Comment:
nit : add the changelog entry in the middle (say line 163) instead of append it in the last line would save some conflict resolve efforts in case any

Reply:
correct the PR link to remove the "file" at the end?
```suggestion
- [Multiple Datasource] Add show default datasource for selector picker ([#6293](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6293))
```

Reply:
+1

Reply:
Got it